pkg/loki/modules.go

```diff
@@ -275,9 +275,9 @@ func (t *Loki) initIngester() (_ services.Service, err error) {
 	if err != nil {
 		return
 	}
-	logproto.RegisterPusherServer(t.Server.GRPC, t.Ingester)
-	logproto.RegisterQuerierServer(t.Server.GRPC, t.Ingester)
-	logproto.RegisterIngesterServer(t.Server.GRPC, t.Ingester)
+	//logproto.RegisterPusherServer(t.Server.GRPC, t.Ingester)
+	//logproto.RegisterQuerierServer(t.Server.GRPC, t.Ingester)
+	//logproto.RegisterIngesterServer(t.Server.GRPC, t.Ingester)
```

Comment:
We services must be registered *after* they were wrapped.

Reply:
This was fixed by adding the wrapper.